### PR TITLE
Bullets have higher chance to hit enemies

### PR DESCRIPTION
**Pull request by GianiStatie**

### CODE REVIEW

File: src/units/00_components/AbstractBullet.gd [hunk 1]
```diff
@@ -3,30 +3,29 @@ class_name Bullet
 
 var speed: int
```
Comment (Ratoone) [inline src/units/00_components/AbstractBullet.gd:4]: i think this speed is no longer used now

File: src/units/00_components/AbstractBullet.gd [hunk 2]
```diff
@@ -3,30 +3,29 @@ class_name Bullet
...
-	global_position = global_position.move_toward(
-		target_enemy_position,
-		delta * speed
-	)
+	global_position += velocity * delta
 
 func _on_projectile_hit(_target: Node2D):
 	queue_free()
+
+
```
Comment (Ratoone) [inline src/units/00_components/AbstractBullet.gd:29]: i think we should enforce some standard about spacing between methods. i know python has 2, but we've been using 1 line in most places, and we want to be consistent

File: src/units/00_components/abstract_bullet.tscn [hunk 3]
```diff
@@ -20,7 +20,7 @@ region_rect = Rect2(352, 192, 32, 32)
 
 [node name="Hitbox" type="Area2D" parent="."]
 collision_layer = 4
-collision_mask = 2
+collision_mask = 10
```
Comment (Ratoone) [inline src/units/00_components/abstract_bullet.tscn:23]: Why does this need to collide with a tower and why doesn't it disappear when it does?

**Review by Ratoone — APPROVED** [review]

File: src/units/00_components/abstract_bullet.tscn [hunk 4]
```diff
@@ -30,5 +30,6 @@ shape = SubResource("CircleShape2D_puen5")
 [node name="Timer" type="Timer" parent="."]
 wait_time = 2.0
 
-[connection signal="area_entered" from="Hitbox" to="." method="_on_projectile_hit"]
+[connection signal="area_entered" from="Hitbox" to="." method="_on_hitbox_area_entered"]
+[connection signal="body_entered" from="Hitbox" to="." method="_on_hitbox_body_entered"]
```
Comment (Ratoone) [inline src/units/00_components/abstract_bullet.tscn:34]: You know you can connect multiple signals to the same function, right?

Comment (GianiStatie) [reply src/units/00_components/abstract_bullet.tscn:34]: oh right... I just clicked mindlessly

**Pull request merged**